config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -45,6 +45,9 @@ presets:
   # Increase throughput in Kubemark master components.
   - name: KUBEMARK_MASTER_COMPONENTS_QPS_LIMITS
     value: "--kube-api-qps=100 --kube-api-burst=100"
+  # Allow one node to not be ready after cluster creation.
+  - name: ALLOWED_NOTREADY_NODES
```

Comment:
This simply should be part of kubemark-common.

Reply:
Do we want to have this feature in every kubemark test? I was thinking about adding this only for tests >= 500 kubemark nodes.

Reply:
I've moved `ALLOWED_NOTREADY_NODES` to kubemark-common. You are right - it makes more sense.

For some reason I thought that smaller cluster should flake less.